### PR TITLE
TRST-M-2

### PR DESCRIPTION
**Pull request by Thomas-Smets**
Also covers TRST-L-5

### CODE REVIEW

File: src/accounts/AccountV1.sol [hunk 1]
```diff
@@ -472,6 +484,7 @@ contract AccountV1 is AccountStorageV1, IAccount {
         )
     {
         inAuction = true;
+        lastActionTimestamp = uint32(block.timestamp);
```
Comment (j-vp) [inline src/accounts/AccountV1.sol:487]: can we create this into a modifier?

Comment (Thomas-Smets) [reply src/accounts/AccountV1.sol:487]: Would be more lines, but is possible yes

Comment (j-vp) [reply src/accounts/AccountV1.sol]: Gtg then

**Review by j-vp — APPROVED** [review]
👍

**Pull request merged**